DotaStats/Modules/MatchInfoModule/View/MatchInfoViewController.swift

```diff
@@ -25,10 +27,21 @@ final class MatchInfoViewController: UIViewController {
         tableView.delegate = self
         tableView.dataSource = self
         tableView.backgroundColor = ColorPalette.mainBackground
+        tableView.translatesAutoresizingMaskIntoConstraints = false
         tableView.separatorStyle = .none
         return tableView
     }()
 
+    private lazy var errorView: ErrorView = {
+        let view = ErrorView()
+        view.alpha = 0
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.isUserInteractionEnabled = true
+        let tapActionHideError = UITapGestureRecognizer(target: self, action: #selector(handleTap(_:)))
```

Comment:
Сейчас название этой переменной говорит о том, что в ней хранится ошибка. Лучше назвать ее примерно так
```suggestion
        let tapActionGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(handleTap(_:)))
```